.github/workflows/preview.yml

```diff
@@ -35,6 +35,7 @@ jobs:
         uses: actions/checkout@v2
         with:
           repository: dita-ot/website
+          ref: feature/platform-selector
```

Comment:
Do not merge this

Reply:
Shall we drop https://github.com/dita-ot/docs/pull/300/commits/667a82d29a234a331a3d3362d6346d3fc0db9855 with an interactive rebase before merging, or would you prefer to squash?